ament_cmake_test/cmake/run_test.py

```diff
@@ -152,7 +155,7 @@ def log(msg, **kwargs):
                     value = env[key] + ';' + env_str
                 else:
                     key, value = separate_env_vars(env_str, 'append-env', parser)
-                log(' - {0}={1}'.format(key, value))
+                log(' - {0}+={1}'.format(key, value))
```

Comment:
This will print e.g.:

```
 - ping+=pong
 - ping+=;pong;peng
```

The `+=` is certainly helpful but it shouldn't include `pong` in the second line.